modules/mirror-registry-remote.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-mirroring-installation-images.adoc
+
+[id="mirror-registry-remote_{context}"]
+= Mirroring on a remote host with mirror registry for {product-title}
+
+Remote mirroring with mirror registry for {product-title}
+
+This procedure introduces how to install Red Hat Quay on a remote host using the `mirror-registry` tool. By doing so, users can create a registry to hold a mirror of {product-title} images.
```

Comment:
```suggestion
This procedure introduces how to install the mirroring on a remote host using the `mirror-registry` tool. By doing so, users can create a registry to hold a mirror of {product-title} images.
```

Reply:
This procedure introduces how to install the mirror registry on a remote host using the `mirror-registry` installer tool. By doing so, users can create a local host registry running on port 443 for the purpose of storing a mirror of {product-title} images.